web/lib/HabitatSidebar.js

```diff
@@ -62,6 +62,19 @@ const NAV_TEMPLATE =
   background-color: var(--color-bg-invert);
   color: var(--color-bg);
 }
+
+.balance-title {
+  padding-left: 1em;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.action-link {
+  font-size: 11px;
```

Comment:
Please use `em` or `rem` units, so that nothing gets out of touch if we modify the font sizes of parent elements in the future 👍